packages/affiliates/libs/bigquery.js

```diff
@@ -12,17 +11,15 @@ module.exports = ({ client } = {}) => {
     logsByContract(contract, start, end = Date.now(), selection = ["*"]) {
       assert(contract, "requires contract");
       assert(start, "requires start");
-      start = moment(start).format("YYYY-MM-DD hh:mm:ss");
```

Comment:
I suspect the issue was in how this is formatted. in the LM scripts it's defined as:

```
argv.dateTime, "YYYY-MM-DD  HH:mm Z"
```

see: https://github.com/UMAprotocol/protocol/blob/a0f038ca961df8e00050a57542228da89294575e/packages/core/scripts/liquidity-mining/FindBlockAtTimeStamp.js#L13

In any case I'm happy with doing it with tmiestamps as well as you've done in your PR

Reply:
Agreed. SGTM